packages/rollups-node/s6-overlay/s6-rc.d/stream-cleanup/script/up.sh

```diff
@@ -1,8 +1,14 @@
-#!/usr/bin/env sh
-
-if [ ! -z ${ENABLE_STREAM_CLEANUP+x} ];then
-    printf "stream-cleanup: ENABLE_STREAM_CLEANUP is set, cleaning redis streams...\n"
-    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-claims;
-    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-inputs;
-    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-outputs;
-fi
+#!/command/with-contenv bash
+set -e
+
+DAPP_CONTRACT_ADDRESS=$(jq -r '.address | ascii_downcase' ${DAPP_CONTRACT_ADDRESS_FILE})
```

Comment:
The previous script was using the `DAPP_CONTRACT_ADDRESS` envvar, instead of the `DAPP_CONTRACT_ADDRESS_FILE`

Reply:
Ok, but what if I want to use the `DAPP_CONTRACT_ADDRESS` variable?
Actually I think it's called `DAPP_ADDRESS`

Reply:
Then we need to fix this.

Reply:
I sent a fixup.

Reply:
`DAPP_ADDRESS` is for rollups-node:1.2

Reply:
I updated it to rollups-node:1.2 already.